arangod/RocksDBEngine/RocksDBEdgeIndex.cpp

```diff
@@ -143,7 +143,8 @@ class RocksDBEdgeIndexLookupIterator final : public IndexIterator {
       coveringBuilder->add(_lastKey);
       coveringBuilder->add(fromTo);
       coveringBuilder->close();
-      cb(docId, coveringBuilder->slice());
+      auto data = SliceArrayCoveringData(coveringBuilder->slice());
```

Comment:
Not sure if we need `SliceArrayCoveringData` here. The edge index stores the edge id as a simple string, and not as vpack.

Reply:
renamed to SliceCoveringData to exclude misleading Array from name